treeherder/etl/classification_loader.py

```diff
@@ -88,3 +100,46 @@ def get_push(self, task_route):
             raise
 
         return push
+
+    def autoclassify_failures(self, failures):
+        try:
+            autoclassified_intermittent = FailureClassification.objects.get(
+                name="autoclassified intermittent"
+            )
+        except FailureClassification.DoesNotExist:
+            logger.error(
+                "FailureClassification named 'autoclassified intermittent' does not exist."
+            )
+            raise
+
+        for group, tasks in failures.items():
+            for task in tasks:
+                # Keeping only the tasks that should be autoclassified
+                if not task.get("autoclassify"):
+                    continue
+
+                try:
+                    bug = Bugscache.objects.get(summary__endswith=f"{group} | single tracking bug")
+                except Bugscache.DoesNotExist:
+                    # No associated Bugzilla bug exists, skipping the autoclassification
```

Comment:
I would log as info here "No bug found for group failure {group}"

Reply:
at first this will be a lot of logging, once rolled out more, about half of the failures will fall into the "non test specific" category.  I am ok with or without logging.